(py3) root@kali:~/battle_tested/battle_tested# python -Xfaulthandler runner.py
Fatal Python error: Segmentation fault

Current thread 0x00007f3428767740 (most recent call first):
  File "/root/battle_tested/battle_tested/ammo.py", line 37 in <genexpr>
  File "/root/battle_tested/battle_tested/ammo.py", line 37 in extract_objects
  File "/root/battle_tested/battle_tested/ammo.py", line 20 in wrapper
  File "/root/battle_tested/battle_tested/ammo.py", line 49 in ammo_from_gc
  File "/root/battle_tested/battle_tested/ammo.py", line 55 in infinite_gc_ammo
  File "runner.py", line 9 in fuzz_generator
  File "runner.py", line 14 in runner
  File "runner.py", line 33 in fuzz_test
  File "runner.py", line 47 in main
  File "runner.py", line 56 in <module>
Segmentation fault
